src/main/docker/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM openjdk:8-jre-alpine
+
+ARG SHADED_JAR
+
+ENV DW_REAPER_SEGMENT_COUNT=200 \
```

Comment:
So, that was just for "Dropwizard" but since the application is effectively REAPER we could just drop the prefix as that is likely clearer.